api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 216**
+
+* [`POST /typing`](/api/set-typing-status): When the `type` of the message
+  being composed is `"stream"`, changed the `to` parameter to accept the
+  ID of the stream in which the message is being typed. Previously, it
+  accepted a single-element list containing the ID of the stream.
```

Comment:
This seems like it should be feature level 215 as well. And maybe it's worth a note just after the bullets for that feature level that "Note that stream typing notifications were not enabled in any Zulip client prior to feature level 215.".

Reply:
Made the changes.

Additionally added a line in `zulip.yaml` as [discussed](https://chat.zulip.org/#narrow/stream/378-api-design/topic/stream.20typing.20indicator/near/1658910):

"Clients shouldn't care about the APIs prior to Zulip 8.0 (feature level 215) for stream typing notifications, as no client actually implemented the previous API for those."